### PR TITLE
updated year in footer to 2017 from 2016

### PR DESCRIPTION
**Pull request by jgajera**

### CODE REVIEW

File: source/layouts/regions/_contentinfo.haml [hunk 1]
```diff
@@ -4,7 +4,7 @@
       %ul
         %li.contentinfo-legal
           :markdown
-            Sass &copy; 2006&ndash;2016 [Hampton Catlin](http://www.hamptoncatlin.com/),
+            Sass &copy; 2006&ndash;2017 [Hampton Catlin](http://www.hamptoncatlin.com/),
```
[the remark this answers is not in the document]
Comment (jgajera) [reply source/layouts/regions/_contentinfo.haml]: I don't know what this means. :(

[the remark this answers is not in the document]
Comment (jina) [reply source/layouts/regions/_contentinfo.haml]: It's the automated linting that I put in place after much of the site was built for code quality. It just says the line is 88 characters out of the suggested 80. If you want to fix it, you can just hit enter after the 2017 (with `[Hampton` name lining up with `Sass`). Do you mind doing that fix real quick and I'll merge?

**Comment by jgajera** [comment]
Done! Thank you!

File: source/layouts/regions/_contentinfo.haml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jgajera) [reply source/layouts/regions/_contentinfo.haml]: @jina I think I've done it now! It added another commit

**Comment by jina** [comment]
thanks. next time the site gets deployed this change will be live. :)

**Pull request merged**